internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -168,7 +175,7 @@ func (*Sequencer) Install(r runtime.Runtime) []runtime.Phase {
 	phases := PhaseList{}
 
 	switch r.State().Platform().Mode() { //nolint:exhaustive
-	case runtime.ModeContainer:
+	case runtime.ModeContainer, runtime.ModeMetalAgent:
```

Comment:
should never get here?

Reply:
yep, was a defensive check, but now removed.